docs/src/pages/components/data-grid/group-pivot/group-pivot.md

```diff
@@ -6,6 +6,178 @@ title: Data Grid - Group & Pivot
 
 <p class="description">Use grouping, pivoting, and more to analyze the data in depth.</p>
 
+## Row grouping [<span class="plan-premium"></span>](https://mui.com/store/items/material-ui-pro/)
+
+Use grouping columns to group the rows according to one or several columns value</p>
```

Comment:
```suggestion
Use row grouping to group the rows according to one or several column values</p>
```
Thanks for changing the name. There are a few more leftovers like this, so I won't try to suggest them all.

Reply:
I forgot most of those on the doc page indeed.
Done :+1: